adal/src/androidTest/java/com/microsoft/aad/adal/MockDeviceCertProxy.java

```diff
@@ -77,12 +72,22 @@ public String getThumbPrint() {
         return sThumbPrint;
     }
 
+    @Override
+    public byte[] sign(String algorithm, byte[] dataToBeSigned) throws ClientException {
```

Comment:
I'm wondering why this change is being needed now

Reply:
Looks like the super class was updated in common, but this class that extends it was not updated. I was getting a complitation error.